### PR TITLE
Min supported version

### PR DESCRIPTION
**Pull request by shabtaisharon**

### CODE REVIEW

File: ds3_java_cli/src/main/java/com/spectralogic/ds3cli/Main.java [hunk 1]
```diff
@@ -32,7 +33,13 @@
     public static void main(final String[] args) {
         try {
             final Arguments arguments = new Arguments(args);
+
             final Ds3Client client = createClient(arguments);
+            if (!Utils.isCliSupported(client)) {
+                System.out.println(String.format("ERROR: Minimum Black Pearl supported is %s", Utils.MINIMUM_VERSION_SUPPORTED));
+                System.exit(2);
```
Comment (DenverM80) [inline ds3_java_cli/src/main/java/com/spectralogic/ds3cli/Main.java]: what does exit status of 2 indicate?

Comment (shabtaisharon) [inline ds3_java_cli/src/main/java/com/spectralogic/ds3cli/Main.java]: that there was an error

Comment (rpmoore) [inline ds3_java_cli/src/main/java/com/spectralogic/ds3cli/Main.java]: Is there a reason it's 2 rather than say 1?

Comment (DenverM80) [inline ds3_java_cli/src/main/java/com/spectralogic/ds3cli/Main.java]: AFAIK, non zero indicates error, so I was curious about 2 vs 1

Comment (shabtaisharon) [inline ds3_java_cli/src/main/java/com/spectralogic/ds3cli/Main.java]: I used the same status code like the other error in that flow (in the end of the file)

Comment (rpmoore) [inline ds3_java_cli/src/main/java/com/spectralogic/ds3cli/Main.java]: Ok.  I couldn't remember the reason why I made a distinction.

**Pull request merged**